app/Http/Controllers/admin/ScholarshipController.php

```diff
@@ -77,13 +77,19 @@ public function edit($id)
   public function update($id)
   {
       $input = Input::all();
-    // Must explicity get checkbox content to save as boolean, since a non-checked  box doesn't return anything from a form submit.
-    $optional = Input::get('display_optional_rec_question');
+
+      // Must explicity get checkbox content to save as boolean, since a non-checked  box doesn't return anything from a form submit.
+      $optional = Input::get('display_optional_rec_question');
+      $images = Input::get('image_uploads');
+
       $scholarship = Scholarship::whereId($id)->firstOrFail();
+
       $scholarship->display_optional_rec_question = $optional;
+      $scholarship->image_uploads = $images;
```

Comment:
You can probably combine the two lines into one:

```php
$scholarship->image_uploads = Input::get('image_uploads');
```
Could also maybe do that with the `$optional` var as well and keep these both below the line where it tries to find or fail on setting `$scholarship`. :)

Reply:
@weerd great call!!